python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -238,30 +242,55 @@ def read_fn():
         self.disable_node_updaters = self.config["provider"].get(
             "disable_node_updaters", False
         )
+        if self.disable_node_updaters:
+            logger.info("Node updaters are disabled.")
```

Comment:
Why not log the flag value regardless of the value? Will it get too noisy?

Reply:
Will do.